ci-operator/step-registry/cucushift/installer/rehearse/nutanix/ipi/flow/provision/cucushift-installer-rehearse-nutanix-ipi-flow-provision-chain.yaml

```diff
@@ -0,0 +1,11 @@
+chain:
+  as: cucushift-installer-rehearse-nutanix-ipi-flow-provision
+  steps:
+  - chain: ipi-conf-nutanix
+  - ref: ipi-conf-additional-ca-trust-policy
+  - chain: ipi-install
+  - ref: configure-registry-storage-nutanix
```

Comment:
The flow cluster profile can not install csi storage by running `ipi-install-nutanix-csi` ?

Reply:
I think so, when using ipi-install-nutanix-csi, I got this error: https://redhat-internal.slack.com/archives/CF8SMALS1/p1733982400428549

Reply:
@sgaoshang did you involve storage team to check the CIS pv provision issues ? Is there any conclusion ?

Reply:
I discussed it with pewang, the collusion is it's not like an storage issue, seems like vpn config issue or auth issue to connect PE, besides nutanix csi driver is not covered by them, so we can workaround it.

Reply:
@sgaoshang  The workaround is using an empty storage pv, right? I think it deserve some bug for tracking, after all, persistent user data is very common and important in the real world, while an empty dir storage can not persist users data. Of course, for now, we can use the empty storage pv as a temp workaround to get this PR merged.

Reply:
Sure, I created a ticket to track it. [SPLAT-1959](https://issues.redhat.com/browse/SPLAT-1959) Investigate Nutanix flow csi driver issue